website/common/script/ops/buy/purchase.js

```diff
@@ -144,8 +144,11 @@ module.exports = function purchase (user, req = {}, analytics) {
     throw new NotAuthorized(i18n.t('notEnoughGems', req.language));
   }
 
-  let itemInfo = getItemInfo(user, type, item);
-  removeItemByPath(user, itemInfo.path);
+  let singlePurchaseTypes = ['gear'];
+  if (singlePurchaseTypes.indexOf(type) !== -1) {
```

Comment:
this line kinda confused me a bit 😁 , this could be converted to `.includes(type)` which is more understandable right away

Reply:
There was another line of code (132-133 purchase.js) which uses a similar style, would you like me to switch that one too? It's unrelated to this issue but it's in the same function.
```
 let acceptedTypes = ['eggs', 'hatchingPotions', 'food', 'quests', 'gear', 'bundles'];
  if (acceptedTypes.indexOf(type) === -1) {
```